jac-uk/admin
# #1758 Batch Archiving

## What's included?
Users who have permissions to update exercises can select multiple exercises and archive them all at once.

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
Go to the Exercises page. When no exercises have been selected the Archive Exercise button should be disabled.
If one or more exercises are selected (using the checkboxes) then the Archive Exercise button should be activated.
When the Archive Exercise button is clicked you should see a modal which allows you to Cancel or Accept the action. If you press Cancel then the modal will disappear. If you press Accept then the modal will disappear and the exercise will be removed from the list.
If you then press the View Archived Exercises button in the top right you should see a list of archived exercises.
The Unarchive Exercise button should be disabled.
If one or more exercises are selected (using the checkboxes) then the Unarchive Exercise button should be activated.
When the Unarchive Exercise button is clicked you should see a modal which allows you to Cancel or Accept the action. If you press Cancel then the modal will disappear. If you press Accept then the modal will disappear and the exercise will be removed from the list.

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
